pom.xml

```diff
@@ -53,11 +54,6 @@
             <artifactId>azure-cosmos</artifactId>
             <version>4.11.0</version>
         </dependency>
-        <dependency>
```

Comment:
did you intentionally remove this dependency?

Reply:
Yeah, I thought it unnecessary to pull in schema-registry here.